redwood-tooling-schema/src/test/kotlin/app/cash/redwood/tooling/schema/SchemaParserTest.kt

```diff
@@ -406,14 +409,14 @@ class SchemaParserTest(
   )
 
   @Test fun scopedChildrenInvalid() {
-    assumeTrue(parser != SchemaParser.Fir)
-
     assertFailure { parser.parse(ScopedChildrenInvalidSchema::class) }
       .isInstanceOf<IllegalArgumentException>()
-      .hasMessage(
-        "@Children app.cash.redwood.tooling.schema.SchemaParserTest.ScopedChildrenInvalidWidget#children lambda receiver can only be a class. " +
-          "Found: kotlin.collections.List<kotlin.Int>",
-      )
+      .message()
+      .isNotNull()
+      .all {
+        contains("@Children app.cash.redwood.tooling.schema.SchemaParserTest.ScopedChildrenInvalidWidget#children lambda receiver can only be a class.")
+        containsMatch(Regex("""Found: (kotlin\.collections\.)?List<(kotlin\.)?Int>"""))
+      }
```

Comment:
Can't easily get these to render the same. Will switch back to testing only one format once we drop reflection.